Refresh.GameServer/Endpoints/Game/Handshake/AuthenticationEndpoints.cs

```diff
@@ -261,18 +262,21 @@ private static void SendPlatformNotAllowedNotification(GameDatabaseContext datab
     /// </summary>
     [GameEndpoint("goodbye", HttpMethods.Post, ContentType.Xml)]
     [MinimumRole(GameUserRole.Restricted)]
-    public Response RevokeThisToken(RequestContext context, GameDatabaseContext database, GameUser user)
+    public Response RevokeThisToken(RequestContext context, GameDatabaseContext database, GameUser user, Token token, MatchService matchService)
     {
-        string? token = context.Cookies["MM_AUTH"];
+        // On LBP1, our concept of a "room" is very basic, and we can be sure that they are the only user in said "room",
+        // so we just remove it outright if the user is logging out
+        // NOTE: We don't need any extra logic to handle more complex rooms here,
+        //       since LBP2 and LBP3 do not actually send requests to this endpoint when closed.
```

Comment:
I think they can, but the games usually segfault before the game can actually send this request. Best to not leave this as a special case. Only condition should be if the user is the host of that room.

Reply:
Even when the games cleanly exited (tried on both PS3 and RPCS3, i could get the games to safely close) i couldn't get either to send this request, but I'll extend the logic to handle it anyway

Reply:
I'd prefer it be correct even if the games are stupid.